sql-cli/sql_cli/__main__.py

```diff
@@ -168,20 +171,34 @@ def run(
     # decide this during runtime, depending on the project path and SQL CLI configuration.
     airflow_meta_conn = retrieve_airflow_database_conn_from_config(project.directory / project.airflow_home)
     set_airflow_database_conn(airflow_meta_conn)
-    dag_file = cli.generate_dag(
-        project=project,
-        workflow_name=workflow_name,
-        env=env,
-        generate_tasks=generate_tasks,
-    )
+    dag_file = _generate_dag(project=project, workflow_name=workflow_name, generate_tasks=generate_tasks)
     dag = get_dag(dag_id=workflow_name, subdir=dag_file.parent.as_posix(), include_examples=False)
     rprint(f"\nRunning the workflow [bold blue]{dag.dag_id}[/bold blue] for [bold]{env}[/bold] environment\n")
-    cli.run_dag(
-        project=project,
-        env=env,
-        dag=dag,
-        verbose=verbose,
-    )
+    try:
+        dr = cli.run_dag(
+            project=project,
+            dag=dag,
```

Comment:
don't we need the `env` parameter anymore?

Reply:
No we only need it for the project.